app/lib/notifications/conversations.rb

```diff
@@ -0,0 +1,30 @@
+module Notifications
```

Comment:
Can you remove this class and the references from the specs instead? 
Because this class is not used anywhere in the code.

Reply:
Hi @AlexAvlonitis tks for the PR review. I've remove the class then. I hope it's all good. Tks. Leandro.

Reply:
perfect!